### PR TITLE
[docs] Update Jamf docs to align with docs guidelines

### PR DESCRIPTION
**Pull request by colleenmcginnis**
## What does this PR do?

Updates the Jamf Compliance Reporter integration docs to align with the documentation guidelines established in https://github.com/elastic/integrations/pull/3433.  

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] review by @jamiehynds 
- [ ] add PR link to changelog
- [ ] fill in example
- [x] run `docs-lint`

## Related issues

- https://github.com/elastic/integrations/pull/3433.

### CODE REVIEW

File: packages/jamf_compliance_reporter/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,46 +1,60 @@
 # Jamf Compliance Reporter
 
-The [Jamf Compliance Reporter](https://docs.jamf.com/compliance-reporter/documentation/Compliance_Reporter_Overview.html) Integration collects and parses data received from Jamf Compliance Reporter using a TLS or HTTP endpoint.
+The Jamf Compliance Reporter integration collects and parses data received from [Jamf Compliance Reporter](https://docs.jamf.com/compliance-reporter/documentation/Compliance_Reporter_Overview.html) using a TLS or HTTP endpoint.
+
+Use the Jamf Compliance Reporter integration to collect logs from your machines.
+Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference data when troubleshooting an issue.
+
+For example, if you wanted to be notified when (something happens) you could (collect data). Then you can (visualize|alert|troubleshoot) by (doing something in Kibana).
```
Comment (colleenmcginnis) [inline packages/jamf_compliance_reporter/_dev/build/docs/README.md:8]: @jamiehynds can you help with a high-level example here?

File: packages/jamf_compliance_reporter/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.2"
+  changes:
+    - description: Update docs to align with new docs guidelines.
+      type: enhancement
+      link: TBD
```
Comment (colleenmcginnis) [inline packages/jamf_compliance_reporter/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/3862
```

File: packages/jamf_compliance_reporter/_dev/build/docs/README.md (hunk 1 above)
Comment (jamiehynds) [reply packages/jamf_compliance_reporter/_dev/build/docs/README.md:8]: For example, if you wanted to monitor shell script commands performed by the root user, you can configure Jamf to monitor those events (see [here](https://docs.jamf.com/compliance-reporter/documentation/Audit_Log_Levels_in_Compliance_Reporter.html)) and send to Elastic for further investigation.

**Comment by colleenmcginnis** [comment]
Thanks @jamiehynds! Once I have a ✅  here I'll publish to the docs site.

**Review by jamiehynds — APPROVED** [review]

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**